distutils/cygwinccompiler.py

```diff
@@ -80,6 +80,15 @@ def get_msvcr():
         elif msc_ver == '1600':
             # VS2010 / MSVC 10.0
             return ['msvcr100']
+        elif msc_ver == '1700':
+            # VS2012 / MSVC 11.0
+            return ['msvcr110']
+        elif msc_ver == '1800':
+            # VS2013 / MSVC 12.0
+            return ['msvcr120']
+        elif int(msc_ver) >= 1900 and int(msc_ver) < 2000:
+            # VS2015 / MSVC 14.0
+           return ['ucrt', 'vcruntime140'] 
```

Comment:
How can I confirm these are the right values to confirm? Have you tested distutils/setuptools with this patch? Does it fix the issue reported in the linked discussion?

Reply:
cc @lazka